CONTRIBUTOR_ROLES.md

```diff
@@ -0,0 +1,224 @@
+# Community participant roles
+
+- [Community participant roles](#community-participant-roles)
+  - [Community Participant](#community-participant)
+    - [Responsibilities](#responsibilities)
+    - [How community participants can get involved with the community](#how-community-participants-can-get-involved-with-the-community)
```

Comment:
Maybe two levels for the table of contents is enough information here?

Reply:
I reckon that makes sense. I have an extension in VSCode that generates this :) I am sure you can configure the level though.

Reply:
I can take this on in a follow-up